test/WebJobs.Script.Tests.Integration/SamplesEndToEndTests.cs

```diff
@@ -1015,6 +1015,8 @@ public SamplesEndToEndTests(TestFixture fixture)
         [Fact]
         public async Task HostLog_AdminLevel_Succeeds()
         {
+            TestHelpers.ClearHostLogs();
```

Comment:
While I was debugging the test, I noticed that it was searching through more logs than it needed to. Making it more efficient.